src/modules/CRUD/components/Details/Edit/Edit.js

```diff
@@ -47,6 +47,7 @@ class Edit extends React.Component {
     history: PropTypes.shape({
       push: PropTypes.func,
     }),
+    t: PropTypes.func.isRequired,
```

Comment:
I don't think setting `t()` as required is a good idea.

Alternative could be in setting a `defaultProp` to `{ t: text => text }`